daemon/stats_unix_test.go

```diff
@@ -0,0 +1,28 @@
+package daemon
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"gotest.tools/v3/assert"
+)
+
+func TestGetSystemCPUUsageParsing(t *testing.T) {
+	dummyFilePath := filepath.Join("testdata", "stat")
+	expectedCpuUsage := uint64(65647090000000)
+	expectedCpuNum := uint32(128)
+
+	origStatPath := procStatPath
+	procStatPath = dummyFilePath
+	defer func() { procStatPath = origStatPath }()
+
+	_, err := os.Stat(dummyFilePath)
+	assert.NilError(t, err)
+
+	cpuUsage, cpuNum, err := getSystemCPUUsage()
+
+	assert.Equal(t, cpuUsage, expectedCpuUsage)
+	assert.Equal(t, cpuNum, expectedCpuNum)
```

Comment:
We can inline `expectedCpuUsage` and `expectedCpuNum` and use `cmp` assertions:

```suggestion
	assert.Check(t, is.Equal(cpuUsage, uint64(65647090000000)))
	assert.Check(t, is.Equal(cpuNum, uint32(128)))
```